include/swift/Serialization/DeclTypeRecordNodes.def

```diff
@@ -163,7 +163,7 @@ OTHER(TOP_LEVEL_CODE_DECL_CONTEXT, 239)
 OTHER(GENERIC_PARAM_LIST, 240)
 TRAILING_INFO(GENERIC_PARAM)
 TRAILING_INFO(GENERIC_REQUIREMENT)
-TRAILING_INFO(GENERIC_ENVIRONMENT)
+OTHER(GENERIC_ENVIRONMENT, 243)
```

Comment:
Ugh, yeah. I want "other but just give me the next ID because I don't care". Also, I seem to have to put `SILGenericEnvironment` into the decls & types set, even though it won't be used there.